### DISCUSSION TITLE
unable to download a video on france.tv: SideDownload error:

### DISCUSSION THREAD

**Discussion by choss38** [aclap-dev/video-downloadhelper#435] [Help and Q&A] [answered]
### Your question

Hi,
I can't download this video: https://www.france.tv/slash/split/saison-1/5424624-prends-ma-main.html

`
SideDownload error:

[h264 @ 0x4d4d8c0] out of range intra chroma pred mode
[h264 @ 0x4d4d8c0] error while decoding MB 0 0
[h264 @ 0x4d4d8c0] corrupted macroblock 2 2 (total_coeff=-1)
[h264 @ 0x4d4d8c0] error while decoding MB 2 2
[h264 @ 0x4d6c480] top block unavailable for requested intra mode -1
[h264 @ 0x4d6c480] error while decoding MB 0 0, bytestream 8113
[h264 @ 0x4d6c480] error while decoding MB 0 0, bytestream -15
[h264 @ 0x4da7500] top block unavailable for requested intra mode -1
[h264 @ 0x4da7500] error while decoding MB 0 0, bytestream 12168
[h264 @ 0x4da7500] error while decoding MB 0 0, bytestream -15
[h264 @ 0x4de6980] top block unavailable for requested intra mode -1
[h264 @ 0x4de6980] error while decoding MB 0 0, bytestream 15369
[h264 @ 0x4de6980] error while decoding MB 1 0, bytestream -24
[h264 @ 0x4e0f0c0] left block unavailable for requested intra4x4 mode -1
[h264 @ 0x4e0f0c0] error while decoding MB 0 0, bytestream 35071
[h264 @ 0x4e0f0c0] error while decoding MB 4 0, bytestream -19
[aac @ 0x4e35c40] skip_data_stream_element: Input buffer exhausted before END element found
[h264 @ 0x4d4d8c0] out of range intra chroma pred mode
[h264 @ 0x4d4d8c0] error while decoding MB 10 1
[h264 @ 0x4d6c480] Reference 3 >= 3
[h264 @ 0x4d6c480] error while decoding MB 0 0, bytestream 52
[h264 @ 0x4da7500] Reference 6 >= 3
[h264 @ 0x4da7500] error while decoding MB 0 0, bytestream 107
[h264 @ 0x4de6980] error while decoding MB 2 0, bytestream -27
[h264 @ 0x4e0f0c0] Reference 3 >= 3
[h264 @ 0x4e0f0c0] error while decoding MB 2 0, bytestream 900
[h264 @ 0x4d4d8c0] cbp too large (3199971767) at 3 10
[h264 @ 0x4d4d8c0] error while decoding MB 3 10
[h264 @ 0x4d6c480] Reference 4 >= 3
[h264 @ 0x4d6c480] error while decoding MB 3 0, bytestream 25
[h264 @ 0x4da7500] top block unavailable for requested intra mode
[h264 @ 0x4da7500] error while decoding MB 2 0, bytestream 19
[h264 @ 0x4de6980] Reference 4 >= 3
[h264 @ 0x4de6980] error while decoding MB 0 0, bytestream 97
[h264 @ 0x4e0f0c0] Reference 3 >= 3
[h264 @ 0x4e0f0c0] error while decoding MB 2 0, bytestream 714
[h264 @ 0x4d4d8c0] cbp too large (3199971767) at 13 1
[h264 @ 0x4d4d8c0] error while decoding MB 13 1
[h264 @ 0x4d6c480] error while decoding MB 0 0, bytestream -22
[h264 @ 0x4da7500] error while decoding MB 3 0, bytestream -17
[h264 @ 0x4de6980] Reference 3 >= 3
[h264 @ 0x4de6980] error while decoding MB 2 0, bytestream -3
[h264 @ 0x4e0f0c0] Reference 3 >= 3
[h264 @ 0x4e0f0c0] error while decoding MB 2 0, bytestream 406
[h264 @ 0x4d4d8c0] corrupted macroblock 8 1 (total_coeff=-1)
[h264 @ 0x4d4d8c0] error while decoding MB 8 1
[h264 @ 0x4d6c480] Reference 3 >= 3
[h264 @ 0x4d6c480] error while decoding MB 0 0, bytestream 23
[h264 @ 0x4da7500] error while decoding MB 4 0, bytestream -10
[h264 @ 0x4de6980] Reference 4 >= 3
[h264 @ 0x4de6980] error while decoding MB 1 0, bytestream 105
[h264 @ 0x4e0f0c0] Reference 4 >= 3
[h264 @ 0x4e0f0c0] error while decoding MB 0 0, bytestream 943
[h264 @ 0x4d4d8c0] negative number of zero coeffs at 5 2
[h264 @ 0x4d4d8c0] error while decoding MB 5 2
[h264 @ 0x4d6c480] Reference 3 >= 3
[h264 @ 0x4d6c480] error while decoding MB 1 0, bytestream 9
[h264 @ 0x4da7500] error while decoding MB 1 0, bytestream -9
[h264 @ 0x4de6980] Reference 3 >= 3
[h264 @ 0x4de6980] error while decoding MB 2 0, bytestream 5
[h264 @ 0x4e0f0c0] Reference 3 >= 3
[h264 @ 0x4e0f0c0] error while decoding MB 2 0, bytestream 484
[mp4 @ 0x4e768c0] Could not find tag for codec none in stream #0, codec not currently supported in container
[out#0/mp4 @ 0x5557300] Could not write header (incorrect codec parameters ?): Invalid argument
Error opening output file /home/anonymous/Téléchargements/Split saison 1 épisode 1 en replay.mp4.
Error opening output files: Invalid argument

`

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: fr
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/anonymous/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/anonymous",
  "converterBinary": "/home/anonymous/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}

**Comment by Mattmjs** [comment, marked as answer]
It's not going to work for your site, this is not an issue with the software but the content being viewed.
This what I saw below, I believe it was saying there was an error because I hadn't enabled DRM in Firefox.
![Screenshot 2024-04-28 122319](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/36c500fb-96e5-4da3-b8d8-8c4d105477b7)

Hit details also showed what I highlighted
![frtv](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/ed8eb915-f9cc-478b-a76c-7b62be0a8085)


If you see this in Firefox then nothing be done unfortunately
![DRM icon](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/738ae5cf-18a9-444b-a75c-975a0338fa1e)

**Comment by choss38** [comment]
Ok, thanks for your answer.
